cmd/dp-deployer/main.go

```diff
@@ -57,53 +40,32 @@ type healthcheckConfig struct {
 func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	log.Namespace = "dp-deployer"
-	flag.Parse()
-
-	h, err := initHandlers(ctx)
-	if err != nil {
-		log.Event(ctx, "failed to initialise handlers", log.FATAL, log.Error(err))
-		os.Exit(1)
-	}
 
-	ec := &engine.Config{
-		ConsumerQueue:    *consumerQueue,
-		ConsumerQueueURL: *consumerQueueURL,
-		ProducerQueue:    *producerQueue,
-		Region:           *region,
-		VerificationKey:  *verificationKey,
-	}
-	e, err := engine.New(ec, h)
+	cfg, err := config.Get()
 	if err != nil {
-		log.Event(ctx, "failed to create engine", log.FATAL, log.Error(err))
+		log.Event(ctx, "Failed to initialise config", log.FATAL, log.Error(err))
 		os.Exit(1)
 	}
 
```

Comment:
now that you have config, can you log it out at startup time - common pattern in our apps:
```suggestion
	log.Event(ctx, "config on startup", log.INFO, log.Data{"config": cfg})
```